composer.json

```diff
@@ -38,13 +38,13 @@
         "netresearch/composer-installers": "*"
     },
     "require": {
-        "php": ">=7.0.0 <7.5",
+        "php": "^7.2 || ^8.0",
         "composer-plugin-api": "^1.0.0 || ^2.0.0"
     },
     "require-dev": {
         "composer/composer": "1.2.*@dev || 2.0.*@dev",
         "overtrue/phplint": "^2.0",
-        "phpunit/phpunit": "^6.5"
+        "phpunit/phpunit": "^6.5 || ^7.5 || ^8.5"
```

Comment:
Do we need to support three versions of phpunit then?
I guess 6.5 at least can/must be dropped

Reply:
I guess we can then drop 7.5 as well: 8.5 supports 7.2-8.0, that should be sufficient.